### PR TITLE
ryelang 0.0.20 (new formula)

### PR DESCRIPTION
**Pull request by stefanb**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/r/ryelang.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %w[
+      -s -w
+    ]
+
+    ENV["CGO_ENABLED"] = "0"
+    system "go", "build", *std_go_args(ldflags:)
```
Comment (SMillerDev) [inline Formula/r/ryelang.rb:17]:
```suggestion
    ENV["CGO_ENABLED"] = "0"
    system "go", "build", *std_go_args(ldflags: "-s -w")
```

Comment (chenrui333) [reply Formula/r/ryelang.rb:17]: ➕

Comment (stefanb) [reply Formula/r/ryelang.rb]: Tnx, done.

**Review by p-linnane — APPROVED** [review]
Looks good, thanks! I'll DM you and explain how you can merge this PR!

**Pull request merged**